### ISSUE TITLE
[Bug] Other players fishing at Jerry Pond are detected as NPCs by NPC check

### ISSUE THREAD

**Issue by ILikePlayingGames** [BiscuitDevelopment/SkyblockAddons#383]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
Other players who are using the fishing rod at Jerry Pond are incorrectly detected as NPCs by the NPC check.

**To Reproduce**
Steps to reproduce the behavior (how did you get this bug to occur):
1. Go to Jerry Pond
2. Players holding fishing rods are hidden

**Expected behavior**
The other players aren't detected as NPCs

**Screenshots**
N/A

**Crash Report**
N/A

**System Information**
 - OS: Windows 10
 - Mod Loader: Forge
 - Minecraft Version/Mod Loader Version: 1.8.9 - 11.15.1.2318
 - SkyblockAddons Version: 1.6.0-beta.4

**Other Installed Mods**
N/A

**Additional info**
Reported by @iHDeveloper

**Comment by iHDeveloper** [comment]
If I find any more details about the issue, I will include them here. 😃

**Comment by iHDeveloper** [comment]
![image](https://user-images.githubusercontent.com/20463031/92445620-48c1e100-f1bd-11ea-8dc4-28053c233e3b.png)

**Comment by iHDeveloper** [comment]
![image](https://user-images.githubusercontent.com/20463031/92445918-ba9a2a80-f1bd-11ea-88d7-b27179b55acd.png)

**Comment by iHDeveloper** [comment]
When fishing **Frozen Steve** in Jerry's rod. The mod considers it as **NPC** and hides all players around it.

**Comment by ILikePlayingGames** [comment]
Oh thanks!

**Comment by iHDeveloper** [comment]
I tried to fish in **Jerry's Pod** after the fix with commit **(47f7e18)**.
Players are still getting invisible. Especially after fishing **Frozen Steve**.

I copied the NBT of the entity to help you find a solution. 😄 
## `Frozen Steve` NBT Entity
```
Class: EntityOtherPlayerMP
Name: Frozen Steve
NBT Data:
{
    HurtByTimestamp: 0,
    Spawns: [],
    SleepTimer: 0s,
    Attributes: [{
        Base: 1500.0d,
        Name: "generic.maxHealth"
    }, {
        Base: 0.0d,
        Name: "generic.knockbackResistance"
    }, {
        Base: 0.23000000417232513d,
        Name: "generic.movementSpeed"
    }, {
        Base: 1.0d,
        Name: "generic.attackDamage"
    }],
    Invulnerable: 0b,
    PortalCooldown: 0,
    AbsorptionAmount: 0.0f,
    abilities: {
        invulnerable: 0b,
        mayfly: 0b,
        instabuild: 0b,
        walkSpeed: 0.1f,
        mayBuild: 1b,
        flying: 0b,
        flySpeed: 0.05f
    },
    FallDistance: 0.0f,
    DeathTime: 0s,
    XpSeed: 0,
    HealF: 1500.0f,
    XpTotal: 0,
    Motion: [-0.006375d, -0.078375d, -0.263d],
    UUIDLeast: -7706956892908065399L,
    Health: 1500s,
    foodSaturationLevel: 5.0f,
    Air: 300s,
    OnGround: 1b,
    Dimension: 0,
    Rotation: [-204.13002f, -22.820215f],
    XpLevel: 0,
    Score: 0,
    UUIDMost: -4293184904562465207L,
    Sleeping: 0b,
    Pos: [-48.04000914494741d, 76.00091449474165d, 50.22423696844994d],
    Fire: 0s,
    XpP: 0.0f,
    EnderItems: [],
    foodLevel: 20,
    foodExhaustionLevel: 0.0f,
    HurtTime: 0s,
    SelectedItemSlot: 0,
    Inventory: [],
    foodTickTimer: 0
}
```

**Comment by ILikePlayingGames** [comment]
This one was confirmed fixed a while ago, I forgot to close it.
